scripts/webpack.base.js

```diff
@@ -31,8 +31,8 @@ module.exports = {
   plugins: [
     new webpack.DefinePlugin({
       __SCROLLBAR_VERSION__: JSON.stringify(
-        process.env.SCROLLBAR_VERSION || require('../package.json').version,
-      ),
+        process.env.SCROLLBAR_VERSION || require('../package.json').version
+      )
```

Comment:
and please exclude changes in this file too 😁

Reply:
Ok for the second comma, but the comma after `version` cause this error:
![image](https://user-images.githubusercontent.com/4406419/33316605-cd90eb00-d434-11e7-86bb-35d21a85d03a.png)

Reply:
So you're running on node <= 6? That would be fine with higher version of node.